src/control/client/pool.go

```diff
@@ -27,30 +27,100 @@ import (
 	"golang.org/x/net/context"
 
 	pb "github.com/daos-stack/daos/src/control/common/proto/mgmt"
+	"github.com/daos-stack/daos/src/control/logging"
+	"github.com/pkg/errors"
 )
 
-// CreatePool will create a DAOS pool using provided parameters and return uuid
-func (c *connList) CreatePool(req *pb.CreatePoolReq) ResultMap {
-	results := make(ResultMap)
-	mc := c.controllers[0] // connect to first AP only for now
+func verifyUnicast(cs []Control) (Control, error) {
```

Comment:
This seems like a strange choice. Wouldn't it be better to name it something like chooseServiceLeader() and just return the first host for now? It could return an error if there are no hosts configured or a future selection algorithm fails, but for now it's very simple. As it is, you're guaranteeing that you'll have to change this function and all of its callers in the future when the code is smarter about finding and using the current management service leader.